base/broadcast.jl

```diff
@@ -1216,6 +1216,7 @@ Base.@propagate_inbounds dotview(args...) = Base.maybeview(args...)
 # broadcasting "dot" calls/assignments:
 
 dottable(x) = false # avoid dotting spliced objects (e.g. view calls inserted by @view)
+dottable(x::Function) = dottable(nameof(x))
```

Comment:
It seems simpler/better to me to just make this `true`. The reason not to dot operators already beginning with a dot is that they're not function calls, but syntax for broadcast. But a call with a spliced function object will always be a function call.